### PR TITLE
Add next_stage spec

### PR DESCRIPTION
**Pull request by driusan**
Add README to next_stage module

### CODE REVIEW

File: modules/next_stage/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Next Stage
+
+## Purpose
+
+The `next_stage` module provides a method of start the "next stage"
```
Comment (PapillonMcGill) [inline modules/next_stage/README.md]: suggest "... method of starting the ..." or "... method to start the ... "

File: modules/next_stage/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+of a candidate session. The "Stage" of an session goes from Screening
```
Comment (PapillonMcGill) [inline modules/next_stage/README.md]: ... of **a** session ...

File: modules/next_stage/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+to Visit to Approval, and the Screening and Visit stages may have
```
Comment (PapillonMcGill) [inline modules/next_stage/README.md]: Approval**. The** Screening ...

File: modules/next_stage/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+test battery for a session.
+
+## Intended Users
+
+The `next_stage` module is used by data entry staff in order to
+start the Visit stage of an instrument.
+
+## Scope
+
+The `next_stage` module only moves from the Screening to Visit stage
```
Comment (PapillonMcGill) [inline modules/next_stage/README.md]: suggest: ... only move the stage of a session from Screening to Visit.

File: modules/next_stage/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+for a session. The final Approval or Recycling Bin stage is reached
+by sending to DCC (or Recycling Bin) on the `instrument_list` page.
+
+## Permissions
+
+Both the `data_entry` permission is required, and the user must be
```
Comment (PapillonMcGill) [inline modules/next_stage/README.md]: Is there 2 permissions required?
one suggestion is : "The `data_entry` permission is required. The user must also be at the same CenterID ..."

or "... The user wil only see the timepoint for CenterID where he is activated" or something similar

File: modules/next_stage/README.md (hunk 1 above)
Comment (driusan) [reply modules/next_stage/README.md]: I went with "to" since it requires less formatting change..

File: modules/next_stage/README.md (hunk 2 above)
Comment (driusan) [reply modules/next_stage/README.md]: done

File: modules/next_stage/README.md (hunk 3 above)
Comment (driusan) [reply modules/next_stage/README.md]: done

File: modules/next_stage/README.md (hunk 5 above)
Comment (driusan) [reply modules/next_stage/README.md]: Yes, both the explicit permission with the permission code, and the same CenterID (which isn't really a permission in the code sense, but affects access) is required.

I updated it to your wording, I think it's more clear.

**Pull request merged**